### PR TITLE
Notes for Grid and TreeList behavior when in edit mode

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/grid/editing/overview.md [hunk 1]
```diff
@@ -277,6 +277,17 @@ There are a few considerations to keep in mind with the CUD operations of the gr
 
     @* Applies to the other input type components as well *@
     ````
+    
+* If you want to perform other data operations while the component is in Edit mode (applicable for [InCell]({%slug components/grid/editing/incell%}) and [Inline]({%slug components/grid/editing/inline%}) editing) the following behavior will occur:
+
+    * For operations like Filter, Group, Sort, Paging, Search, Select, Row drag and Delete:
```
Comment (dimodi) [inline components/grid/editing/overview.md:283]: Use the same grammatical form:

```suggestion
    * For operations like Filter, Group, Sort, Page, Search, Select, Row drag and Delete:
```

File: components/grid/editing/overview.md [hunk 2]
```diff
@@ -277,6 +277,17 @@ There are a few considerations to keep in mind with the CUD operations of the gr
...
+        * InCell edit -  if the validation is satisfied, a save operation will be executed. If the validation is **not** satisfied, edit will be cancelled with `OnCancel` event and `IsCancelled` will be considered.
```
Comment (dimodi) [inline components/grid/editing/overview.md:284]: What does it mean "`IsCancelled` will be considered"?? 

```suggestion
        * InCell edit -  if the validation is satisfied, a save operation will be executed. If the validation is **not** satisfied, editing will be cancelled with `OnCancel` event and `IsCancelled` will be considered.
```

File: components/grid/editing/overview.md [hunk 3]
```diff
@@ -277,6 +277,17 @@ There are a few considerations to keep in mind with the CUD operations of the gr
...
+        * Inline edit - regardless of the validation, the edit will be cancelled with `OnCancel` event and `IsCancelled` will be considered.
```
Comment (dimodi) [inline components/grid/editing/overview.md:285]: Same as above

File: components/grid/editing/overview.md [hunk 4]
```diff
@@ -277,6 +277,17 @@ There are a few considerations to keep in mind with the CUD operations of the gr
...
+
+    * For operations like Edit, Add, Save:
+        * InCell edit - if the validation is satisfied, the currently edited item will be saved and the command will be executed. If the validation is **not** satisfied, the command will be blocked until the item is valid or editing is cancelled.
+        * Inline edit - if the validation is satisfied, `OnCancel` will be fired for the currently edited item the command will be executed. If the validation is **not** satisfied, the command will be blocked until the item is valid or editing is cancelled.
```
Comment (dimodi) [inline components/grid/editing/overview.md:289]: Looks like there are missing words here, or leftovers from a previous sentence edit.

File: components/treelist/editing/overview.md [hunk 5]
```diff
@@ -380,6 +380,16 @@ There are a few considerations to keep in mind with the CUD operations of the tr
 @* Applies to the other input type components as well *@
 ````
 
+* If you want to perform other data operations while the component is in Edit mode (applicable for [InCell]({%slug treelist-editing-incell%}) and [Inline]({%slug treelist-editing-inline%}) editing) the following behavior will occur:
```
Comment (dimodi) [inline components/treelist/editing/overview.md:383]: Consider a _contentTemplate for this repeating content.

**Review by dimodi — CHANGES_REQUESTED** [review]
TreeList / Line 289 needs rewording. Consider a reusable _contentTemplate.

File: components/treelist/editing/overview.md (hunk 5 above)
Comment (ntacheva) [reply components/treelist/editing/overview.md:383]: The links in this exact paragraph are different (one points to the corresponding articles for grid and the other - for treelist). However, I may use a template for the rest of the content.

File: components/grid/editing/overview.md (hunk 4 above)
Comment (ntacheva) [reply components/grid/editing/overview.md:289]: Fixed

File: components/treelist/editing/overview.md (hunk 5 above)
Comment (marin-bratanov) [reply components/treelist/editing/overview.md:383]: You can consider using relative links rather than slugs to put this in a template.

**Review by marin-bratanov — APPROVED** [review]

**Review by dimodi — APPROVED** [review]

**Pull request merged**